src/server/check-answers/index.njk

```diff
@@ -0,0 +1,129 @@
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% extends 'layouts/questions.njk' %}
+
+{% macro section(rows) %}
+
+  {% set enrichedRows = [] %}
+  {% for row in rows %}
+    {% set rows = (enrichedRows, enrichedRows.push({
+      key: {
+        text: row.key,
+        classes: "govuk-!-width-one-half govuk-!-font-weight-regular"
+      },
+      value: {
+        text: row.value
+      },
+      actions: {
+        items: [
+          {
+            href: row.url,
+            text: "Change",
+            visuallyHiddenText: question,
+            attributes: attributes
+          }
+        ]
+      }
+    })) %}
+  {% endfor %}
+
+
+
+  {{ govukSummaryList({
+    rows: enrichedRows
+  }) }}
+{% endmacro %}
+
+{% block beforequestion %}
+
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-body" data-testid="app-page-body">
+        <div class="govuk-!-margin-top-0 govuk-!-margin-bottom-8">
+          {% if (errorMessage) %}
+            {{ govukErrorSummary({
+              titleText: "There is a problem",
+              errorList: [
+                {
+                  text: errorMessage.text,
+                  href: "#confirmation"
+                }
+              ]
+            }) }}
+          {% endif %}
+          <h1 class="govuk-heading-l">
+              {{ heading }}
+          </h1>
+
+          <h2 class="govuk-heading-m">Movement origin</h2>
+          {{ section([{
+            key: "Are you moving the cattle on or off your farm or premises?", 
+            value: "", 
+            url: "/origin/summary"
```

Comment:
The urls here should take you to the question you want to change directly, rather than the summary page.

Reply:
and the summary page should take you to the first question it finds that is invalid

Reply:
thus meaning we dont have to duplicate logic

Reply:
This isn't about invalid pages - you could have valid data you wish to change.  e.g. your CPH may be valid, but you may realise it's the wrong one and click on 'change'